Common/src/grid_movement_structure.cpp

```diff
@@ -4578,6 +5288,58 @@ void CSurfaceMovement::SetHicksHenne(CGeometry *boundary, CConfig *config, unsig
   
 }
 
+void CSurfaceMovement::SetSurface_Bump(CGeometry *boundary, CConfig *config, unsigned short iDV, bool ResetDef) {
```

Comment:
Exactly, there are some design problem where we just want to add a bump in a surface (not an airfoil with chord=1) . Currently we are using HH but in the future we should define this in a more general sense, including 3D (for now, it is a place holder for future improvements).